todocli/task.py

```diff
@@ -0,0 +1,56 @@
+from datetime import datetime
+from enum import Enum
+
+from todocli.todo_api_util import api_timestamp_to_datetime
+
+
+class Task:
+    class Status(Enum):
+        Completed = "completed"
+        NotStarted = "notStarted"
+        InProgress = "inProgress"
+        WaitingOnOthers = "waitingOnOthers"
+        Deferred = "deferred"
+
+    class Importance(Enum):
+        Low = "low"
+        Normal = "normal"
+        High = "high"
+
+    def __init__(self, query_result_task):
+        self.title: str = query_result_task["title"]
+        self.id: str = query_result_task["id"]
+        self.importance: Task.Importance = Task.Importance(
+            query_result_task["importance"]
+        )
```

Comment:
Then this would change to
```suggestion
        self.importance = Task.Importance[query_result_task["importance"]]
```
I'm assuming `query_result_task["importance"]` is a string.

Reply:
I changed `self.importance: Task.Importance = Task.Importance(...)` to `self.importance = Task.Importance(...)` as I agree that there is no need in explicitly specify the type here.